ts/navigation/MainNavigator.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import ROUTES from './routes'
+
+import { Icon } from 'native-base'
+import { TabBarBottom, TabNavigator } from 'react-navigation'
+import MessagesScreen from '../screens/main/MessagesScreen'
+import ProfileScreen from '../screens/main/ProfileScreen'
+import PortfolioNavigator from './PortfolioNavigator'
+
+interface IRouteIcon
+{
+  routeName: string,
+  routeIcon: string
+}
+
+const ROUTE_ICON: ReadonlyArray<IRouteIcon> = [
+  { routeName: ROUTES.MAIN_MESSAGES, routeIcon: "mail" },
+  { routeName: ROUTES.PORTFOLIO_HOME, routeIcon: "wallet" },
+  { routeName: ROUTES.DOCUMENTS_HOME, routeIcon: "document" },
+  { routeName: ROUTES.PREFERENCES_HOME, routeIcon: "cog" },
+  { routeName: ROUTES.MAIN_PROFILE, routeIcon: "user" }
+]
+
+const getIcon = (routeName: string) => {
```

Comment:
`routeName: keyof typeof ROUTES` (you probably want to define a type alias for `keyof typeof ROUTES`)

cc @fpersico